### ISSUE TITLE
Drive A unmounted on CTRL+ALT+DEL

### ISSUE THREAD

**Issue by ShanuJackal** [nbarkhina/DosWasmX#8]
When I restart the DOS prompt with CTRL+ALT+DEL, it forgets that A: exists and unmounts the drive.

**Comment by nbarkhina** [comment]
@ShanuJackal thanks for letting me know, I will take a look

**Comment by nbarkhina** [comment]
Yes there is currently an issue where the A drive does not persist after restart. However you should still be able to remount it after. Run the following commands:

```
mount e .
e:
dir
imgmount a DISK01.IMG
```
The `dir` command is to display the file names. You should see your disk file there as an example lets say its called DISK01.IMG. This should now bring you back to the A drive with the disk mounted. In terms of the message that says "C Drive does not exist" you can ignore this and just run `c:` you will see the C drive is still there.

**Comment by ShanuJackal** [comment]
Well, I have another problem. I'm trying to install MS-DOS 6.22, but it's telling me I have to restart to begin setup. You see the loop I'm stuck in. What do I do? (try it yourself, i got the disk files from WinWorld https://winworldpc.com/download/c38fc38d-68c2-bbe2-80a6-4b11c3a4c2acc )

**Comment by ShanuJackal** [comment]
Oop just one more thing

There's no way to eject a floppy, MS-DOS installation requires ejecting setup disk at the end of install
